source/module_io/istate_charge.h

```diff
@@ -27,16 +25,23 @@ class IState_Charge
     void begin(Gint_Gamma& gg,
                elecstate::ElecState* pelec,
                const ModulePW::PW_Basis* rhopw,
-               const ModulePW::PW_Basis_Big* bigpw);
+               const ModulePW::PW_Basis_Big* bigpw,
+               const bool gamma_only_local,
+               const int nbands_istate,
+               const int nbands,
+               const double nelec,
+               const int nspin,
+               const std::string& global_out_dir,
+               const int my_rank,
+               std::ofstream& ofs_warning);
 
   private:
-    int *bands_picked;
```

Comment:
these two ways to declare one pointer is not a relationship "right or wrong", a casual discussion can be find such like https://stackoverflow.com/questions/6990726/correct-way-of-declaring-pointer-variables-in-c-c.
More generally, it might be `int * bands_picked` (but I am not meaning to suggest to change to this).

Reply:
I am using automatic formatting plugin in VSCode, so the pointer usage style is default. I mainly followed this Feishu Doc written by @haozhihan: https://xmywuqhxb0.feishu.cn/docx/QpBrdRKrDohEzXxO4TQcwOO0njg